product_docs/docs/pgd/5/manual/01-provisioning-hosts.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Provisioning Hosts
+---
+
+## Provisioning hosts
+
+The first step in the process of deploying PGD is to provision and configure hosts.
+
+You can deploy to virtual machine instances in the cloud with Linux installed, on-premise virtual machines with Linux installed or on-premise physical hardware also with Linux installed. 
+
+Whichever [supported Linux operating system](https://www.enterprisedb.com/resources/platform-compatibility#bdr) and whichever deployment platform you select, the result of provisioning a machine should be a Linux system that can be accessed by you using SSH.
+
+Each machine provisioned should be able to make connections to any other machine you are provisioning for your cluster. 
+On cloud deployments, this may be done over the public network or over a VPC. 
+On-premise deployments should be able to connect over the local network.
+
+!!! Note Cloud provisioning guides
+
+If you are new to cloud provisioning, these guides may provide assistance:
+
+  Vendor | Platform | Guide
+  ------ | -------- | ------
+  Amazon | AWS      | [Tutorial: Get started with Amazon EC2 Linux instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EC2_GetStarted.html)
+  Microsoft | Azure | [Quickstart: Create a Linux virtual machine in the Azure portal](https://learn.microsoft.com/en-us/azure/virtual-machines/linux/quick-create-portal?tabs=ubuntu)
+  Google | GCP | [Create a Linux VM instance in Compute Engine](https://cloud.google.com/compute/docs/create-linux-vm-instance)
+
```

Comment:
This table renders weird, likely because of the `!!!` below?
![image](https://github.com/EnterpriseDB/docs/assets/263427/49c50ea7-9a02-478d-b7a5-946adb6daf52)

Reply:
Further further investigation shows that the system sets a css class of table-with-scroll on a containered table so that will need some investigation.